src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1228,6 +1228,9 @@
         - name: --disable-advanced-network-observability
           type: bool
           short-summary: Disable advanced network observability functionalities on a cluster
+        - name: --advanced-networking-observability-tls-management
```

Comment:
This seems to be the help msg for `az aks update`, please also add the info for `az aks create`.

Reply:
Updated and removed help messages in _params.py